src/select/ValueGroup.js

```diff
@@ -8,7 +8,7 @@ class ValueGroup extends Component {
     inputValue: PropTypes.string,
     labelKey: PropTypes.string,
     onMouseDown: PropTypes.func,
-    placeholder: PropTypes.string,
+    placeholder: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
That's a good point since that's exactly how I used it.  For example:
  renderPlaceholder = () => {
    const { label } = this.props
    return (<span><FA name="clock-o" className="placeholder-icon" />{label}</span>)
  }

What do you think? Change it to function or string instead of object or string?

Reply:
Yes, let's do that. Sounds like `PropTypes.oneOfType([PropTypes.string, PropTypes.func])` would be more accurate.